x-pack/plugins/licensing/public/plugin.ts

```diff
@@ -76,18 +77,16 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup> {
     );
     this.licenseFetchSubscription = fetchSubscription;
 
-    const license$ = update$.pipe(
-      tap(license => {
```

Comment:
The laziness tripped me up. We didn't store license and re-fetch a new one because there are no subscribers to license$.